goodkey/good_key.go

```diff
@@ -76,7 +76,8 @@ func NewKeyPolicy(weakKeyFile, blockedKeyFile string) (KeyPolicy, error) {
 
 // GoodKey returns true if the key is acceptable for both TLS use and account
 // key use (our requirements are the same for either one), according to basic
-// strength and algorithm checking.
+// strength and algorithm checking. GoodKey only supports reference rsa.PublicKey
+// and ecdsa.PublicKey and will reject non-reference versions.
```

Comment:
```suggestion
// strength and algorithm checking. GoodKey only supports pointers: *rsa.PublicKey
// and *ecdsa.PublicKey. It will reject non-pointer types rsa.PublicKey and ecdsa.PublicKey.
```